autosave/mixins.py

```diff
@@ -196,13 +195,14 @@ def response_change(self, request, obj):
         return response
 
     def render_change_form(self, request, context, add=False, obj=None, **kwargs):
-        if 'media' in context:
-            get_params = u''
-            if 'is_retrieved_from_autosave' in request.POST:
-                get_params = u'?is_recovered=1'
-            autosave_media = self.autosave_media(obj, get_params=get_params)
-            if isinstance(context['media'], basestring):
-                autosave_media = unicode(autosave_media)
-            context['media'] += autosave_media
+        if not add:
```

Comment:
Might as well combine this conditional with the following one so that the indentation doesn't have to change in the diff, e.g. `if not add and 'media' in context:`